README.md

```diff
@@ -89,6 +89,20 @@ This config option controls the export file format for each Sphinx builder. It
 accepts a dictionary mapping builder names to image formats. Accepted values for
 the latter are `"png"`, `"jpg"`, or `"svg"`.
 
+### Output directory
+- *Formal Name*: `drawio_export_directory`
+- *Default Value*: `None`
+
+This config option controls where the generated files are exported to.
+Both relative(relative to the current working directory) and absolute paths
```

Comment:
```suggestion
Both relative (to current working directory) and absolute paths
```
Uh, is this the current working directory? What use does that have, should this not be relative to say, the sphinx build directory?

Reply:
The use of this could be if you run sphinx from a build system (html and latex one after another) and you want to share for example .png rendered drawio diagrams.
In this case the build directories of the two sphinx runs are not predictable.
This just adds the option to share the rendered diagrams between these sphinx runs to save the rendering time.
In make based build systems paths relative to the working directory are handy in some cases.

To be honest I just want the output of the plugin to be inside the output directory of each builder (the current implementation places them in the source directory). The intention behind adding this option was only to support the rare case of sphinx builds with multiple builders using the same output format with lots of drawio graphs to save build time...

If you do not like this option at all I do not have any problem to discard this idea - as long as the default output directory is inside the output directory ;)

Reply:
I see no problem with either option... if no one actually wants it (do you?) I don't see the point of maintaining the extra code as opposed to the sensible default.

Also; your (recent commit's) change still didn't have space between the paranthesis and text.

Reply:
I added the space (sorry, I overlooked this one)

...I would keep the option. It is very little code and it might be useful for someone.
Initially I added it because changing the default behavior might increase the built time for certain users. This option shall prevent breaking their workflow...

Reply:
I would like to keep this option, but resolving relative paths with respect to the current working directory sounds like trouble. I think it makes most sense to interpret them relative to the _conf.py_ directory. If someone wants more "freedom", they can always supply an absolute path.

Note that options can be overridden from the "sphinx-build" command-line, so one doesn't have to edit _conf.py_ to change this.